file.js

```diff
@@ -21,15 +21,15 @@ document.querySelectorAll('.link').forEach((n) => n.addEventListener('click', ()
 const myProjects = [
```

Comment:
- [ ] As requested by the previous reviewer, kindly follow the given design, the bottom buttons are not clearly visible. You could add a scroll property on the popup so users can scroll all the way to the bottom of the page.

Your design
![image](https://user-images.githubusercontent.com/81932478/180458454-473e91e5-7de0-47f1-af52-1bc35bafa89a.png)

Figma template
![image](https://user-images.githubusercontent.com/81932478/180458878-f9f245ac-e55c-42b1-8c33-3217b056a80f.png)

- [ ] Also the positioning of the items in your design is not as is in the Figma template, kindly fix this. If you look at the image it is not centered.
- [ ] The languges list needs some padding so that they are not so close to each other
![image](https://user-images.githubusercontent.com/81932478/180461639-790f2f1e-c5ed-4e29-8007-f0783b8306f0.png)

- [ ] Bottom buttons are not properly aligned.

Your design
![image](https://user-images.githubusercontent.com/81932478/180462200-66f2b269-4eb2-4818-b262-42d1205e3265.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/180463401-33c06774-9811-4b4c-bc8a-67d97a013c1a.png)


- [ ] The typography and color of the text have not been addressed as well, kindly fix this issue.

Your design
![image](https://user-images.githubusercontent.com/81932478/180462765-3a5149ca-6845-47af-8615-aeb90499affd.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/180463909-2dde3665-3c5a-4d22-aa4a-34590341f328.png)